app/src/main/java/com/sdp13epfl2021/projmag/MainActivity.kt

```diff
@@ -16,6 +16,8 @@ class MainActivity : AppCompatActivity() {
     companion object MainActivityCompanion {
         const val fromLinkString: String = "fromLink"
```

Comment:
A comment or doc explaining these values could be great for the future

Reply:
add comment in https://github.com/asfoury/projmag/pull/123/commits/287d1c2db39db8253c368377ff32c58eb5f3ede1